profiles/aom.js

```diff
@@ -59,23 +53,6 @@ const modules = [
   import("../src/core/linter-rules/no-http-props.js"),
 ];
 
-async function domReady() {
-  if (document.readyState === "loading") {
-    await new Promise(resolve =>
-      document.addEventListener("DOMContentLoaded", resolve)
-    );
-  }
-}
-
-(async () => {
-  const [runner, { ui }, ...plugins] = await Promise.all(modules);
-  try {
-    ui.show();
-    await domReady();
-    await runner.runAll(plugins);
-  } finally {
-    ui.enable();
-  }
-})().catch(err => {
-  console.error(err);
-});
+Promise.all(modules)
+  .then(plugins => ReSpec.run(plugins))
+  .catch(err => console.error(err));
```

Comment:
I would've liked `await ReSpec.run(modules)`, but that would mean passing `moduleImportPromises` to `ReSpec.run`, which feels awkward.
Also, top-level await isn't common enough yet I think?

Reply:
Yeah, once Safari 15 is out we can switch 🥳